content/php-client/resources/reference-entity/media-files.md

```diff
@@ -0,0 +1,23 @@
+### Reference entity media file
+
+::: warning
+This resource is only available in the [Entreprise Edition](https://www.akeneo.com/enterprise-edition/).
+:::
+
+#### Download the media file associated to a reference entity or a record
+
+```php
+$client = new \Akeneo\PimEnterprise\ApiClient\AkeneoPimEnterpriseClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+$mediaFileResponse = $client->getReferenceEntityMediaFileAPi()->download('images/kartell.png');
+
+file_put_contents('/tmp/kartell.png', $mediaFileResponse->getBody()->getContents());
```

Comment:
also, it's a code when you try to dowload something, not a path as clean as that.
We should provide an example using the mime type for the response and using the filename from headers.